src/com/company/PlayerBattingStatus.java

```diff
@@ -0,0 +1,17 @@
+package com.company;
+
+public enum PlayerBattingStatus {
```

Comment:
its good that you used enum. but think if it is really necessary here. If a player has batted or not can be determined from the order of batsman playing and wickets fallen. so status1 is not needed. And if a player is out or not can be kept as a boolean flag